OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationOptionSpinner.java

```diff
@@ -23,23 +28,51 @@ public RegistrationOptionSpinner(Context context, AttributeSet attrs, int defSty
         super(context, attrs, defStyle);
     }
 
-    public boolean hasValue(String value){
-        if ( value == null || items == null )
-            return false;
-        for ( RegistrationOption option : items ) {
-            if ( value.equals(option.getValue()) )
-                return true;
+    public boolean hasValue(@Nullable String value){
+        boolean ret = false;
+        if (adapter != null) {
+            ret = (getAdapterPosition(value) >= 0);
+        }
+        return ret;
```

Comment:
@mdinino Could you use more descriptive variable names?

https://www.cs.umd.edu/~clin/MoreJava/Intro/var-descr.html